examples/gbt_cls_model_create_from_lightgbm_batch.py

```diff
@@ -0,0 +1,92 @@
+#*******************************************************************************
+# Copyright 2014-2020 Intel Corporation
+# All Rights Reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License"), the following terms apply:
+#
+# You may not use this file except in compliance with the License.  You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#*******************************************************************************
+
+# daal4py Gradient Bossting Classification model creation from LightGBM example
+
+import daal4py as d4p
+import lightgbm as lgb
+import numpy as np
+
+# let's try to use pandas' fast csv reader
+try:
+    import pandas
+    read_csv = lambda f, c=None, t=np.float64: pandas.read_csv(f, usecols=c, delimiter=',', header=None, dtype=t)
+except:
+    # fall back to numpy loadtxt
+    read_csv = lambda f, c=None, t=np.float64: np.loadtxt(f, usecols=c, delimiter=',', ndmin=2, dtype=t)
+
+def main(readcsv=read_csv, method='defaultDense'):
+    # Path to data
+    train_file = "./data/batch/df_classification_train.csv"
+    test_file = "./data/batch/df_classification_test.csv"
+
+    # Data reading
+    X_train = readcsv(train_file, range(3), t=np.float32)
+    y_train = readcsv(train_file, range(3, 4), t=np.float32)
+    X_test = readcsv(test_file, range(3), t=np.float32)
+    y_test = readcsv(test_file, range(3, 4), t=np.float32)
+
+    # Datasets creation
+    lgb_train = lgb.Dataset(X_train, y_train, free_raw_data=False)
+
+    # training parameters setting
+    params = {
+        'max_bin':          256,
+        'scale_pos_weight': 2,
+        'lambda_l2':        1,
+        'alpha':            0.9,
+        'max_depth':        8,
+        'num_leaves':       2**8,
+        'verbose':          -1,
+        'objective':        'multiclass',
+        'learning_rate':    0.3,
+        'num_class':        5,
+    }
+
+    # Training
+    lgb_model = lgb.train(params, lgb_train, valid_sets=lgb_train)
+
+    # LightGBM prediction
+    lgb_prediction = lgb_model.predict(X_test)
+    lgb_errors_count = np.count_nonzero(np.argmax(lgb_prediction, axis=1) - np.ravel(y_test))
```

Comment:
I see you use translate `np.argmax(lgb_prediction, axis=1)` to labels more one. Maybe use variable?

Reply:
For interest. Did you compare the probabilities?

Reply:
1 - done
2 - No, we didn't. I think, that complete accuracy match might be considered as a good indicator